cli.py

```diff
@@ -0,0 +1,282 @@
+# Created by Vijay Rajagopal
+import os
+from src.evaluators.tube_evaluator import TubeEvaluator
+import sys
+import argparse
+import logging
+from src.utils import converter
+from src.utils.enumerators import (BBFormat, BBType, CoordinatesType,
+                                   MethodAveragePrecision)
+import src.evaluators.coco_evaluator as coco_evaluator
+import src.evaluators.pascal_voc_evaluator as pascal_voc_evaluator
+import src.utils.converter as converter
+import src.utils.general_utils as general_utils
+from src.bounding_box import BoundingBox
+import matplotlib.pyplot as plt
+
+def parseArgs():
+    parser = argparse.ArgumentParser()
+    # Folder path to annotations
+    parser.add_argument('--anno_gt', type=str)
+    parser.add_argument('--anno_det', type=str)
+
+    # Folder path to corresponding images:
+    parser.add_argument('--img', type=str, required=False)
+    #parser.add_argument('--img_gt', type=str, required=False)
+    #parser.add_argument('--img_det', type=str, required=False)
+
+    # format_gt: as in the application screenshot:
+    parser.add_argument('--format_gt', type=str)
+
+    # format_det: shares input from coord (either xyrb, xywh, or coco)
+    parser.add_argument('--format_det', type=str)
+
+    # Absolute or relative (abs, rel)
+    #parser.add_argument('--gtcoord',  type=str)
+    parser.add_argument('--coord_det', type=str)
+
+    # Actual computation type:
+    parser.add_argument('--metric', type=str)
+    
+    # metadata for metrics (not always needed)
+    parser.add_argument('--names', '-n', type=str, default='')
+    parser.add_argument('-t', '--threshold', type=float, default=0.5)
+    
+    # extra data (graphs and etc.)
+    parser.add_argument('--prgraph', '-pr', action='store_true')
```

Comment:
popular cli tools usually call it `--plot`

Reply:
Fixed in 780b32cd8054d602ec7d9679ae0f4dcc9e4d8943